sycl/test/basic_tests/event.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: opencl
+// REQUIRES: opencl || level0
```

Comment:
I wonder how clCreateUserEvent call below will be processed if only Level0 RT is available

Reply:
clCreateUserEvent is not called for host and the test sets SYCL_DEVICE_TYPE=HOST. Do you suggest separating this test too?

Reply:
I am not quite understand the logic of the test:
 1. if we force HOST device lines 26-31 are dead code.
 2. It looks like the whole test check host events. What is the reason for having OpenCL or Level0 BE?